sourcegraph/sourcegraph
# Render "no data" error message for searches with no results

Closes https://github.com/sourcegraph/sourcegraph/issues/40871

display "zero data" error message

| Design | Actual |
| --- | --- |
| ![PixelSnap 2022-09-02 at 13 07 44](https://user-images.githubusercontent.com/1855233/188212864-5b3b37ed-2215-430e-8367-ef61d223c407.png)|![PixelSnap 2022-09-02 at 13 11 49](https://user-images.githubusercontent.com/1855233/188213430-08c9d4e1-c057-41c1-ae35-9eada27819d4.png)|

## Test plan

1. Run any search that returns no results. For example "asdfasdfasdf"
2. Should hide the aggregations panel completely.
3. Search for "repo:sourcegraph/sourcegraph" 
4. Should render the chart
5. Change group by to "File"
6. Should render the "No data to display" over the chart skeleton

## App preview:

- [Web](https://sg-web-insights-zero-data-chart-display.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-hdsctkgyac.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
